client/packages/common/src/ui/forms/JsonForms/components/CategorizationTabLayout.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { FC, useState } from 'react';
+import {
+  and,
+  Categorization,
+  categorizationHasCategory,
+  Category,
+  isVisible,
+  LayoutProps,
+  optionIs,
+  RankedTester,
+  rankWith,
+  uiTypeIs,
+} from '@jsonforms/core';
+import { withJsonFormsLayoutProps } from '@jsonforms/react';
+import { Button, Grid } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { DialogButton } from '../../../components/buttons';
+import { ModalProps, useDialog } from '@common/hooks';
+import {
+  AjvProps,
+  MaterialLayoutRenderer,
+  MaterialLayoutRendererProps,
+  withAjvProps,
+} from '@jsonforms/material-renderers';
+
+interface CategoryModalProps extends ModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const CategoryModal: FC<CategoryModalProps> = ({
+  children,
+  isOpen,
+  onClose,
+  ...modalProps
+}) => {
+  const { Modal } = useDialog({
+    isOpen,
+    onClose,
+  });
+  return <Modal {...modalProps}>{children}</Modal>;
+};
+
+export const categorizationTabLayoutTester: RankedTester = rankWith(
+  2,
+  and(
+    uiTypeIs('Categorization'),
+    categorizationHasCategory,
+    optionIs('variant', 'tab')
```

Comment:
maybe we have to be more specific and call it "bubble-tab"? since its not a normal tab interface...